lib/evss/service.rb

```diff
@@ -36,13 +36,25 @@ def headers_for_user(user)
       EVSS::AuthHeaders.new(user).to_h
     end
 
+    def save_error_details(error)
+      Raven.extra_context(
```

Comment:
Not sure if you all have had the same experiences, but I have found that the best way to conduct efficient searches in Sentry's UI is through their **tagging** fields.  Consequently, I've always made sure to include a unique/searchable `tags_context` param to all of my Sentry logging.

For example, for a senty log that we pass in this to the `tags_context` param:

```
profile: 'pciu_profile'
```

We would be able to run this search query in Sentry and have it immediately return the relevant results:

```
is:unresolved profile:pciu_profile
```

I raise this b/c I personally have not found the `extra_context` to be as accurate, or even usable for searching at times.  Perhaps we should consider using the `tags_context` here, as well.

Reply:
Why not both? 😄I think it depends what we're using the data for, extra context seems to be debug info for specific events while tags are for grouping events. Sentry docs mention that extra context is not indexed:
> These are not indexed and are simply used to add additional information about what might be happening

If we could also have tags that group events to a service and a service action that would help with querying.

Reply:
+1 for both!